README.md

```diff
@@ -37,13 +37,23 @@ Fingerprint Pro Angular SDK is an easy way to integrate  **[Fingerprint Pro](htt
 
 ## Table of contents
 
+- [Requirements](#requirements)
 - [Installation](#installation)
 - [Getting started](#getting-started)
 - [Caching strategy](#caching-strategy)
 - [Documentation](#documentation)
 - [Support and feedback](#support-and-feedback)
 - [License](#license)
 
+## Requirements
+
+The following dependencies are required:
+
+- TypeScript 4
```

Comment:
I presume it's a minimal TypeScript version, right? Or is it not usable with TypeScript 5.x?

```suggestion
- TypeScript 4+
```

Reply:
Going to check with Typescript 5 if it's working.